client/dashboard/emails/index.tsx

```diff
@@ -28,7 +29,10 @@ function Emails() {
 	const { data: filteredData, paginationInfo } = filterSortAndPaginate( emails, view, emailFields );
 
 	return (
-		<PageLayout header={ <PageHeader title={ __( 'Emails' ) } /> }>
+		<PageLayout
+			header={ <PageHeader title={ __( 'Emails' ) } /> }
```

Comment:
Unrelated to this PR, but since we're updating this line here, we can omit the `title={ ( __( 'Emails' ) }` now since `PageHeader` will just extrapolate the title based on the router's meta. See https://github.com/Automattic/wp-calypso/pull/106007.

Reply:
Whoa TIL. done be13a896acefec82245af0a15b74f14a7abc6098